.github/workflows/deploy.yml

```
@@ -0,0 +1,56 @@
+name: Deploy docs site to GitHub Pages
+
+on:
+  # Runs on pushes targeting the `master` branch.
+  push:
+    branches: [master]
```

This is minor but i prefer this syntax:

```suggestion
    branches:
      - master
```

---

Fixed in https://github.com/jchavarri/melange-fest/pull/6/commits/c032c76365f69a86677c09aca0f6d60f9206aaaf